projects/canopy/src/lib/forms/input/input.scss

```diff
@@ -4,8 +4,8 @@
   display: block;
   border: solid var(--border-width) var(--border-color);
   border-radius: var(--border-radius-sm);
-  line-height: var(--line-height-sm);
-  padding: var(--space-xs) var(--space-sm);
+  line-height: var(--input-line-height);
```

Comment:
Would this need to be done also for the select style?

Reply:
Ah, that's a good point, I'll look into that 👍